templates/packer/ansible/roles/common/files/worker-initialization-scripts/vector.sh

```diff
@@ -13,7 +15,7 @@ exclude_units = ["vector.service"]
 [sinks.out]
 type = "aws_cloudwatch_logs"
 inputs = [ "journald" ]
-endpoint = "${CLOUDWATCH_LOGS_ENDPOINT_URL}"
+region = "${REGION}"
```

Comment:
Can you bring back the `endpoint`? It will be useful when we will move internal workers to AWS.

It's weird that vector requires `region` because `endpoint` arguably encodes everything that they need but I guess they had reasons for making it required. :shrug:

Reply:
It's just for the healthcheck, i think it's a bug. I filed it upstream in any case.